ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -446,36 +449,124 @@ def test_full_backup_reinstall_restore_with_vault(self):
 
 
 class TestBackupAndRestoreWithReplica(IntegrationTest):
-    """Regression test for https://pagure.io/freeipa/issue/7234"""
-    num_replicas = 1
+    """
+        Regression tests for issues 7234 and 7455
+
+        https://pagure.io/freeipa/issue/7234
+            - check that oddjobd service is started after restore
+            - check new replica  setup after restore
+        https://pagure.io/freeipa/issue/7455
+            check that after restore and replication reinitialization
+                - users and CA data are at state before backup
+                - CA can be installed on existing replica
+                - new replica with CA can be setup
+    """
+    num_replicas = 2
     topology = "star"
+    csr = "my.csr"
```

Comment:
This will create a file at the current working directory (which can change) and never clean up the file. You need to create a temp file on each node and remove the files in the instance or class tear down hook. See ``upload_temp_contents``.